scripts/scr_inventory_draw.gml

```diff
@@ -29,4 +27,15 @@ if (active)
             }
         }
     }
+    // Highlight selected item if any item exists
+    if (currentItemCount > 0)
+    {
+        draw_set_color(c_yellow);
```

Comment:
do you have to set the color back to what it was before? I'm not sure how drawing works in GML, but does this change it to yellow for everything til it gets updated again? Just making sure.

Reply:
I think this is the only place where I'm settings the colors at the moment, so impact. 

Ideally, the drawer should define the color he/she wants to use right before drawing them, so should be non-issue.